cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -99,6 +99,7 @@ def handle_grading_policy_changed(sender, **kwargs):
         'course_key': unicode(kwargs.get('course_key')),
         'event_transaction_id': unicode(get_event_transaction_id()),
         'event_transaction_type': unicode(get_event_transaction_type()),
+        'countdown': 1800,  # 30 minute countdown
```

Comment:
:) This is a kwarg of the celery task, not the underlying function, so it has to get passed in as it's own keyword argument, not as an entry in the dictionary passed in as the keyword argument `kwargs`.